specification/columns/commitmentutilizationtotal.md

```diff
@@ -0,0 +1,44 @@
+# Commitment Utilization Total
+
+Commitment Utilization Total is a numeric value representing the aggregate unit amount of a purchased [*commitment-based discount*](#glossary:commitment-based-discount) for a billing period.  A CommitmentUtilizationTotal value MUST remain a constant value for a specific [*commitment-based discount*](#glossary:commitment-based-discount) within a billing period. The ratio of [*CommitmentUtilizationUsed*](#commitment-utilization-used) over CommitmentUtilizedTotal produces the utilization rate for a given commitment-based discount or aggregate of.
+
+CommitmentUtilizationTotal column MUST be present in the billing data when the provider supports *commitment-based discounts*.
+
+A positive CommitmentUtilizationTotal value MUST be applied to a [*row*](#glossary:row) when:
+
+* ChargeCategory is *Usage*.
+* ChargeSubCategory is *UsedCommitment* or *UnusedCommitment*.
+* ResourceId is not null.
+* CommitmentDiscountId is not null.
+* CommitmentUtilizationUsed is not null.
+* CommitmentUtilizationUnit is not null.
+
+When the corresponding CommitmentUtilizationUnit is *Spend*, the CommitmentUtilizationTotal value describes a monetary amount matching the associated BillingCurrency.
```

Comment:
Should we instead say:

"The unit for the CommitmentUtilizationTotal is described by the CommitmentUtilizationUnit column. For example, when the commitment utilization unit is US Dollars, the CommitmentUtilizationTotal must be provided in US Dollars."

Reply:
The next sentence "Otherwise, .. MUST be null." makes it sound like this column only support spend.

Reply:
Do you like a generic "Spend" value that also corresponds with the chosen BillingCurrency ... which could be derived as US dollars?